back/internal/usersvc/internal/userdb/mysql/0002_setup_role_table.sql

```diff
@@ -0,0 +1,10 @@
+-- +migrate Up
```

Comment:
Please run `go generate` from within the `userdb/mysql` directory in order to generate the migrations from the sql files.

Reply:
New go.mod and go.sum files have been generated.